components/ProfileMenu.vue

```diff
@@ -34,64 +44,98 @@ const logout = async () => {
+                >
+                  <span
+                    aria-hidden="true"
+                    :class="
+                      profileStore.autoplay
+                        ? 'translate-x-full'
+                        : 'translate-x-0'
+                    "
+                    class="pointer-events-none inline-block aspect-square w-4 transform rounded-full bg-white-1 shadow-lg ring-1 ring-black-separator transition duration-200 ease-in-out"
+                  />
+                </Switch>
+              </button>
             </MenuItem>
-            <MenuItem v-slot="{ active }" as="div">
+            <MenuItem v-slot="{ active }">
               <button
-                :class="active ? 'bg-violet-500 text-white' : 'text-gray-900'"
-                class="w-full px-2 py-2 text-left hover:bg-[red]"
+                :class="{
+                  'bg-label-separator dark:bg-label-dark-separator': active,
+                }"
+                class="w-full rounded-lg px-3 py-2 text-left"
                 @click.stop
               >
-                {{ $t("profile.theme") }}
+                <p>{{ $t("profile.theme") }}</p>
+                <span class="text-label-2 dark:text-label-dark-2">
+                  {{
+                    $colorMode.value === "light"
+                      ? $t("profile.theme-light")
+                      : $t("profile.theme-dark")
+                  }}
+                </span>
               </button>
             </MenuItem>
             <MenuItem v-slot="{ active }" as="div">
               <button
-                :class="active ? 'bg-violet-500 text-white' : 'text-gray-900'"
-                class="w-full px-2 py-2 text-left hover:bg-[red]"
+                :class="{
+                  'bg-label-separator dark:bg-label-dark-separator': active,
+                }"
+                class="w-full rounded-lg px-3 py-2 text-left"
                 @click.stop
               >
-                {{ $t("profile.app-language") }}
+                <p>{{ $t("profile.app-language") }}</p>
+                <span class="text-label-2 dark:text-label-dark-2">
+                  {{ languageName }}
+                </span>
               </button>
             </MenuItem>
             <MenuItem v-slot="{ active }" as="div">
               <button
-                :class="active ? 'bg-violet-500 text-white' : 'text-gray-900'"
-                class="w-full px-2 py-2 text-left hover:bg-[red]"
+                :class="{
+                  'bg-label-separator dark:bg-label-dark-separator': active,
+                }"
+                class="w-full rounded-lg px-3 py-2 text-left"
                 @click.stop
               >
-                {{ $t("profile.content-language") }}
+                <p>{{ $t("profile.content-language") }}</p>
+                <span class="text-label-2 dark:text-label-dark-2">
+                  {{ contentLanguages.join(", ") }}
```

Comment:
I know locales and translation is hard to get right 😅 Please read https://stackoverflow.com/questions/39237769/rtl-languages-comma-separated-lists and use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat or similar.

Not required for this being approved by me though, just a last touch to get it well 🤗